docs-v2/snippets/pre-built-tooling.mdx

```diff
@@ -0,0 +1,168 @@
+export const PreBuiltTooling = ({ provider }) => {
+    const authTypeMap = {
+        OAUTH1: "OAuth",
+        OAUTH2: "OAuth",
+        OAUTH2_CC: "OAuth",
+        BASIC: "Basic",
+        API_KEY: "API Key",
+        APP_STORE: "Custom",
+        BILL: "Bill",
+        SIGNATURE: "Signature",
+        JWT: "JWT",
+        TWO_STEP: "Two Step",
+        TABLEAU: "Tableau",
+    }
```

Comment:
If you generate all the mdx why not use this value this is also defined in the script?

Reply:
Oops, this is a leftover file from when I was trying to reuse the component. Unfortunately Mintlify's build system couldn't handle much complexity in a snippet because it inlines everything in the build process, so there's no way to have a snippet import our data efficiently.